services/configuration/src/transactions.rs

```diff
@@ -117,14 +119,25 @@ fn enough_votes_to_commit(snapshot: &dyn Snapshot, cfg_hash: &Hash) -> bool {
     let schema = Schema::new(snapshot);
     let votes = schema.votes_by_config_hash(cfg_hash);
     let votes_count = votes.iter().filter(|vote| vote.is_consent()).count();
-    let majority_count = match actual_config.majority_count {
+
+    let config: ConfigurationServiceConfig = get_service_config(&actual_config);
+
+    let majority_count = match config.majority_count {
         Some(majority_count) => majority_count as usize,
         _ => State::byzantine_majority_count(actual_config.validator_keys.len()),
     };
 
     votes_count >= majority_count
 }
 
+fn get_service_config(config: &StoredConfiguration) -> ConfigurationServiceConfig {
```

Comment:
Such code looks very strange. Why we have to access to the service by name?

Reply:
Service config is obtained from `services: BTreeMap<String, serde_json::Value>`, that's why we use service name.